sql/src/test/java/io/crate/analyze/UpdateAnalyzerTest.java

```diff
@@ -557,6 +558,62 @@ public void testUpdateElementOfObjectArrayUsingParameterExpressionInsideFunction
             )));
     }
 
+    @Test
+    public void test_update_returning_with_asterisk() {
```

Comment:
I think the test method names could be improved upon a bit further by also stating the expected outcome. 

`test_xy` or `test_that_xy_works`  is always a bit vague. What behavior do we want to see?

e.g. 

`test_update_returning_with_asterisk_contains_all_columns_in_returning_clause`

Reply:
👍